README.md

```diff
@@ -1 +1,16 @@
-# matrix_check_sum
\ No newline at end of file
+# Matrix: Check sum
+Checks if the sum of values in a given row matches the sum of values in the corresponding column.
+
+## Exercise
+* Design and implement a method that checks that for the given matrix, where number of rows are equal to number of columns whether the sum of each row matches the sum of the corresponding column i.e. sum
```

Comment:
I know this isn't very helpful, but I feel like this explanation is difficult to understand. Could we split it out into multiple sentences?

Reply:
Now reworded.